backend/danswer/document_index/vespa/index.py

```diff
@@ -686,14 +711,13 @@ def admin_retrieval(
             # needed for highlighting while the N-gram highlighting is broken /
             # not working as desired
             + f'or ({{defaultIndex: "{CONTENT_SUMMARY}"}}userInput(@query)))'
-            + _build_vespa_limit(num_to_retrieve)
         )
 
         params: dict[str, str | int] = {
             "yql": yql,
             "query": query,
             "hits": num_to_retrieve,
-            "num_to_rerank": 10 * num_to_retrieve,
+            "offset": 0,
```

Comment:
Set all these as 0 just so we remember later how to do it if we decide to introduce pagination